readme.md

```diff
@@ -22,6 +24,10 @@ a roots extension for inlining your CSS properties into the style attribute in a
     extensions: [ rootsInlineCss(options) ]
   ```
 
+  This extension provides a way for you to inline your html with css. By default all of your `html` files will be inlined however you can specify through options which files you want inlined either with a single file or a [glob expression]( https://github.com/isaacs/minimatch).
+
+  A full list of options can be found [here](https://github.com/jonkemp/inline-css).
+
 ### License & Contributing
 
 - Details on the license [can be found here](LICENSE.md)
```

Comment:
this snuck in, but this link is broken because it's `license.md` (all downcase) can you update that as well